hack/.kazelcfg.json

```diff
@@ -0,0 +1,4 @@
+{
+	"GoPrefix": "k8s.io/repo-infra",
+	"AddSourcesRules": false
```

Comment:
we don't need to run kazel if we're not adding the sources rules - it's not providing any value.

Reply:
Some repos, like test-infra, will. The script should be the same